tests/log_fidelity/test_bgp_shutdown.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import pytest
+
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+
+logger=logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+expected_log = "admin state is set to 'down'"
+defined_action = "sudo config bgp shutdown all"
+
+def test_bgp_shutdown(duthosts, rand_one_dut_hostname):
+    duthost=duthosts[rand_one_dut_hostname]
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="bgp_shutdown")
+    loganalyzer.expect_regex=[]
```

Comment:
```suggestion
    loganalyzer.expect_regex=[expected_log]
```

Since we're looking for a specific message we can just list it here instead of using append.